pkg/core/config/tenant.go

```diff
@@ -175,6 +176,14 @@ type NexmoConfiguration struct {
 	From      string `msg:"FROM" envconfig:"NEXMO_FROM" json:"FROM"`
 }
 
+type AuthHook struct {
```

Comment:
This should be `Webhook` or `Hook` instead of `AuthHook`, because the same structure is used across all gear.